droidlet/lowlevel/hello_robot/remote/remote_hello_robot_ros.py

```diff
@@ -0,0 +1,232 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+"""
+# python -m Pyro4.naming -n <MYIP>
+import select
+import logging
+import os
+import json
+import time
+import copy
+from math import *
+from rich import print
+import Pyro4
+import numpy as np
+from droidlet.lowlevel.hello_robot.remote.utils import transform_global_to_base, goto
+from stretch_ros_move_api import MoveNode as Robot
+from droidlet.lowlevel.pyro_utils import safe_call
+
+
+Pyro4.config.SERIALIZER = "pickle"
+Pyro4.config.SERIALIZERS_ACCEPTED.add("pickle")
+Pyro4.config.ITER_STREAMING = True
+
+
+# #####################################################
+@Pyro4.expose
+class RemoteHelloRobot(object):
```

Comment:
Seems like a good amount of duplicate code here

Reply:
maybe we should just create a base class for RemoteHelloRobot?

Reply:
will open an issue for it.